kustomize/components/container-images-registry/README.md

```diff
@@ -0,0 +1,28 @@
+# Update the container registry of the Online Boutique apps
+
+By default, Online Boutique's services' container images are pulled from a public container registry (`gcr.io/google-samples/microservices-demo`). One best practice is to have these container images in your own private container registry. The Kustomize variation in this folder can help with using your own private container registry.
+
+## Change the default container registry via Kustomize
+
+To automate the deployment of Online Boutique integrated with your own container registry, you can leverage the following variation with [Kustomize](../..).
+
+From the `kustomize/` folder at the root level of this repository, execute this command:
+```
+REGISTRY=my-registry # Example: gcr.io/my-project/my-directory
+sed -i "s|CONTAINER_IMAGES_REGISTRY|${REGISTRY}|g" components/container-images-registry/kustomization.yaml
```

Comment:
**Rationale for my change:**
I've changed `"s/CONTAINER_IMAGES_REGISTRY/$REGISTRY/g"` to `"s|CONTAINER_IMAGES_REGISTRY|$REGISTRY|g"` because the `/`s inside `gcr.io/my-project/my-directory` was breaking the `sed` command. So we're no longer using regexes (using `|`s) to avoid `/`s being interpreted as special characters.